babyai/levels/iclr19_levels.py

```diff
@@ -93,10 +63,44 @@ def gen_mission(self):
 
 
 class Level_GoToRedBallNoDists(Level_GoToRedBall):
+    """
+    Go to the red ball. No distractors present.
+    """
+
     def __init__(self, seed=None):
         super().__init__(room_size=8, num_dists=0, seed=seed)
 
 
+class Level_GoToObj(RoomGridLevel):
```

Comment:
What's the difference between `GoToObj` and `GoToRedBallNoDists`?

Reply:
🤔 the difference is that in GoToObj the object is not always a red ball. Small difference, but it will affect sample efficiency.